examples/java-matter-controller/java/src/com/matter/controller/commands/pairing/PairingCommand.java

```diff
@@ -69,7 +161,7 @@ public PairingCommand(
     this.mFilterType = filterType;
 
     try {
-      this.mRemoteAddr = new IPAddress(InetAddress.getByName("0.0.0.0"));
+      this.mRemoteAddr = new IPAddress(InetAddress.getByName("127.0.0.1"));

```

Comment:
Use a constant.

Why are we using ipv4? Why local loopback? should we accept some command line argument instead here?

Reply:
Good catch, we should be able to ping any device with specified IP address

Reply:
mRemoteAddr can be override in other command which involve "ETHERNET". We will implement that command later. just update it with ipv6 address, thanks
      case ETHERNET:
        addArgument("setup-pin-code", 0, 134217727, mSetupPINCode, null);
        addArgument("discriminator", (short) 0, (short) 4096, mDiscriminator, null);
        addArgument("device-remote-ip", mRemoteAddr);
        addArgument("device-remote-port", (short) 0, Short.MAX_VALUE, mRemotePort, null);
        break;